.github/workflows/acquia_cms_cron.yml

```diff
@@ -321,36 +321,10 @@ jobs:
       matrix:
         orca-job:
           - ISOLATED_TEST_ON_CURRENT
-          - INTEGRATED_TEST_ON_LATEST_LTS
+          - INTEGRATED_TEST_ON_PREVIOUS_MINOR
```

Comment:
Why we have removed `INTEGRATED_TEST_ON_LATEST_LTS` and using `INTEGRATED_TEST_ON_PREVIOUS_MINOR` ?

Reply:
As per our discussion, drupal don't have LTS hence we are using previous_minor.